pkg/util/version/const.go

```diff
@@ -8,6 +8,7 @@ var GitCommit = "unknown"
 // InstallStream describes stream we are defaulting to for all new clusters
 var InstallStream = Stream{
 	Version:    NewVersion(4, 4, 17),
+	Channel:    "stable-4.4",
```

Comment:
think these changes can be reverted for now ...

Reply:
same as above